packages/babel-core/src/errors/rewrite-stack-trace.ts

```diff
@@ -70,6 +70,18 @@ function CallSite(filename: string): CallSite {
   } as CallSite);
 }
 
+// https://github.com/nodejs/node/pull/38215/files#diff-670bf55805b781d9a3579f8bca9104c04d94af87cc33220149fd7d37b095ca1cR210
+function isErrorStackTraceLimitWritable() {
+  const desc = Object.getOwnPropertyDescriptor(Error, "stackTraceLimit");
+  if (desc === undefined) {
+    return Object.isExtensible(Error);
+  }
+
+  return Object.prototype.hasOwnProperty.call(desc, "writable")
+    ? desc.writable
+    : desc.set !== undefined;
```

Comment:
```suggestion
  return desc && Object.prototype.hasOwnProperty.call(desc, "writable") && desc.writable;
```

Do you think this would be enough? Since `stackTraceLimit` is only supported by V8, and:
- if the property doesn't exist, there is no reason to set it because we are in a different engine
- in V8 it's a data property, not a setter

Reply:
> in V8 it's a data property, not a setter

Is it possible for the user to set it to have no `setter`?

Reply:
Yes, but it disables `.stack` generation:
```
➜ node
Welcome to Node.js v20.0.0.
Type ".help" for more information.
> Error.stackTraceLimit = 2
2
> new Error().stack
'Error\n    at REPL8:1:1\n    at Script.runInThisContext (node:vm:122:12)'
> let limit = 3;
undefined
> Object.defineProperty(Error, "stackTraceLimit", { get: () => limit, set: v => limit = v })
[Function: Error] { stackTraceLimit: [Getter/Setter] }
> Error.stackTraceLimit
3
> new Error().stack
undefined
> 
```